.github/workflows/release.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         include:
           - os: windows-2022
             target: x86_64-pc-windows-msvc
-          - os: ubuntu-22.04
+          - os: ubuntu-20.04
```

Comment:
20.04 runner is going to be removed by GH in 10 days, we can't go back to it

Reply:
Unfortunate, and the new musl build (thank you everyone who worked on it!) will fix the problem. 

Could you possibly do a backport of the PR and patch to release a musl build of 0.20.0? I would prefer to not have to setup a proper rust build env from source if musl is going to be officially supported. 

If it is too big a task for one release, that is completely fine, and regardless thanks for this awesome project.